.github/test-spec.yml

```diff
@@ -543,3 +543,10 @@
   - "samples/suit/**/*"
   - "subsys/mgmt/suitfu/**/*"
   - "tests/subsys/suit/**/*"
+
+"CI-test-low-level":
+  - "include/event_manager_proxy.h"
+  - "subsys/event_manager_proxy/**/*"
+  - "samples/ipc/**/*"
+  - "samples/event_manager_proxy/**/*"
```

Comment:
This is in scope of apps pipeline. Do you want to duplicate that or it is tested in other way?

Reply:
Low level is using twister to check sample - which is different than apps.
Beside that, this will enable checking IPC at nrf54*.
Beside that, low-level will use it to calculate code coverage of IPC driver.